cranelift/wasm/src/environ/spec.rs

```diff
@@ -587,6 +587,21 @@ pub trait FuncEnvironment: TargetEnvironment {
     fn use_x86_pmaddubsw_for_dot(&self) -> bool {
         false
     }
+
+    ///
```

Comment:
Just making a note here of empty doc-comments to fill in before merge (I know you're working on this at the moment).